src/styles/main.scss

```
@@ -1,3 +1,55 @@
+@import"variables";
+
+html {
+  font-family: "Arial", "Helvetica", sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  margin: 35px auto;
+  width: $calendarWidth;
+  padding: $calendarPadd;
+  gap: $dayGap;
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: $daySize;
+    width: $daySize;
+    border: 1px solid #000;
+    background-color: #eee;
+    transition: all transform 0.5s;
```

Sorry my mistake, this is not working, use this:
```suggestion
    transition: color 0.5s, transform 0.5s;
```